autometa/validation/datasets.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2021 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+pulling data from google drive dataset with simulated or synthetic communities
+"""
+
+
+import gdown
+import os
+import sys
+import logging
+import pandas as pd
+
+from autometa.common.utilities import internet_is_connected
+
+logger = logging.getLogger(__name__)
+
+
+def download(
+    community_type: str, community_sizes: list, file_names: list, dir_path: str
+) -> None:
+
+    """Downloads the files specified in a dictionary.
+
+    Parameters
+    ----------
+    community_type : str
+        specifies the type of dataset that the user would like to download from
+    community_sizes : list
+        specifies the size of dataset that the user would like to download
+    file_names : list
+        specifies the file(s) that the user would like to download
+    dir_path : str
+        output path where the user wants to download the file(s)
+
+    Returns
+    -------
+    None
+        download is completed through gdown
+
+    """
+
+    if community_type == "synthetic" or community_type == "all":
+        raise NotImplementedError
+
+    df = pd.read_csv("gdown_fileIDs.csv", dtype=str, index_col=["dataset", "file"])
```

Comment:
I'm not sure whether this should be hardcoded here. If this is hardcoded here, would it be worth specifying the github URL or google drive URL for this file instead of the file on the user's system? 

1. If the script has made it to this stage, it should be able to connect.
2. This should resolve any issues of the script not being able to find the appropriate path to this file.

Reply:
Oh, I didn't even test that! Good catch. I think you're right, it should redirect out somewhere we know the file is. Do you think github or google drive would be more appropriate? I am thinking google drive would be best since the url would never have to change. It should be a small enough file that we can just use pd.read_csv() directly, without any gdown. (However, I don't have permission to add things to the simulated communities folder where I think this file should be kept.)